plugin-gradle/README.md

```diff
@@ -188,7 +188,7 @@ spotless {
     greclipse().configFile('greclipse.properties')
   }
   groovyGradle {
-    // same as groovy, but for .gradle (defaults to '*.gradle')
+    // same as groovy, but for .gradle (defaults to '**/*.gradle')
```

Comment:
I think the default is `*.gradle`

https://github.com/diffplug/spotless/blob/6c7db83068bd9bf92f6570b3f0845cbaf25aef5d/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/GroovyGradleExtension.java#L24